seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/SeaTunnelServer.java

```diff
@@ -253,4 +257,63 @@ private void printExecutionInfo() {
             coordinatorService.printJobDetailInfo();
         }
     }
+
+    public SeaTunnelConfig getSeaTunnelConfig() {
+        return seaTunnelConfig;
+    }
+
+    public NodeEngineImpl getNodeEngine() {
+        return nodeEngine;
+    }
+
+    public ConnectorPackageService getConnectorPackageService() {
+        int retryCount = 0;
+        if (isMasterNode()) {
+            // The hazelcast operator request invocation will retry, We must wait enough time to
+            // wait the invocation return.
+            String hazelcastInvocationMaxRetry =
+                    seaTunnelConfig
+                            .getHazelcastConfig()
+                            .getProperty("hazelcast.invocation.max.retry.count");
```

Comment:
Please use reference not hard code config. You can refer https://github.com/apache/seatunnel/pull/5618